core/templates/pages/topics-and-skills-dashboard-page/modals/unassign-skill-from-topics-modal.component.html

```diff
@@ -4,26 +4,41 @@ <h3>
   </h3>
 </div>
 <div class="modal-body">
-  <div><em>The topics with this skill as their only diagnostic test skill are not eligible for the unassignment.</em></div><br>
-  <span *ngIf="topicsAssignmentsAreFetched && eligibleTopicsForUnassignment.length !== 0">This skill is assigned to these following topics. Tick the topics that you want this skill to be unassigned from.</span>
+  <div *ngIf="topicNamesNotEligibleForUnassignment.length > 0">
+    <div>
+      <p>This skill is used as the only diagnostic test skill in the following topics:</p>
+      <p>{{topicNamesNotEligibleForUnassignment.join(', ')}}</p>
```

Comment:
Use bullet lists (ul, li elements) that open the corresponding topic editor page in a new tab.

Reply:
Added